coursemapper-kg/recommendation/app/services/course_materials/recommendation/recommendation_type.py

```
@@ -3,9 +3,45 @@
-    STATIC_DOCUMENT_BASED = "static_document_based"
-    STATIC_KEYPHRASE_BASED = "static_keyphrase_based"
-    DYNAMIC_KEYPHRASE_BASED = "dynamic_keyphrase_based"
-    DYNAMIC_DOCUMENT_BASED = "dynamic_document_based"
+    CONTENT_BASED_DOCUMENT_VARIANT = "static_document_based"
+    CONTENT_BASED_KEYPHRASE_VARIANT = "static_keyphrase_based"
+    PKG_BASED_KEYPHRASE_VARIANT = "dynamic_keyphrase_based"
+    PKG_BASED_DOCUMENT_VARIANT = "dynamic_document_based"
     COMBINED_DYNAMIC = "combined_dynamic"
     COMBINED_STATIC = "combined_static"
+
+    def map_type(recommendation_type: str, find_type = "k"):
```

Ruff (0.12.2)

Missing return type annotation for public function `map_type`

(ANN201)